Shared/Features/Freaks/ViewModel/FreaksViewModel.swift

```diff
@@ -23,7 +23,9 @@ final class FreaksViewModel: ObservableObject {
     }
 
     func filterFreaks() {
-        if selectedSkills.isEmpty || selectedProjects.isEmpty {
+        if (selectedSkills.isEmpty || selectedProjects.isEmpty) ||
```

Comment:
So many rules here :D Won't it be the same as resetting the freaks list before every filtering?

Reply:
Yeah I was thinking the same. Should I remove the check?